src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/acs_client.py

```diff
@@ -57,9 +58,14 @@ def secure_copy(user, host, src, dest, key_filename=None, allow_agent=True):
     keys = _load_keys(key_filename, allow_agent)
     pkey = keys[0]
     ssh = paramiko.SSHClient()
+    conf = paramiko.SSHConfig()
+    ssh_config_file = os.path.expanduser("~/.ssh/config")
```

Comment:
What will happen if "~/.ssh/config" is absent ?
Can you make sure it is the same behavior as before ?

Reply:
yes ... i tested it and it works just as before on my devices.

Reply:
cool, i will sign off.

Reply:
On my phone, no approve button😒